hw7/hw3.py

```diff
@@ -0,0 +1,51 @@
+"""
+Given a Tic-Tac-Toe 3x3 board (can be unfinished).
+Write a function that checks if the are some winners.
+If there is "x" winner, function should return "x wins!"
+If there is "o" winner, function should return "o wins!"
+If there is a draw, function should return "draw!"
+If board is unfinished, function should return "unfinished!"
+
+Example:
+    [[-, -, o],
+     [-, x, o],
+     [x, o, x]]
+    Return value should be "unfinished"
+
+    [[-, -, o],
+     [-, o, o],
+     [x, x, x]]
+
+     Return value should be "x wins!"
+
+"""
+from itertools import chain
+from typing import List
+
+import numpy as np
+
+
+def tic_tac_toe_checker(board: List[List]) -> str:
```

Comment:
who wins [["-", "-", "o"], ["-", "o", "o"], ["x", "x", "x"]]?

Reply:
my code shows 'wins x'. 
It is not right?